build/tailscale/files/network-tailscale.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0"?>
+<!DOCTYPE service_bundle SYSTEM "/usr/share/lib/xml/dtd/service_bundle.dtd.1">
+<service_bundle type="manifest" name="export">
+  <service name="ooce/network/tailscale" type="service" version="0">
+    <create_default_instance enabled="true"/>
+    <single_instance/>
+    <dependency name="network" grouping="require_all" restart_on="error" type="service">
+      <service_fmri value="svc:/milestone/network:default"/>
+    </dependency>
+    <dependency name="filesystem" grouping="require_all" restart_on="error" type="service">
+      <service_fmri value="svc:/system/filesystem/local"/>
+    </dependency>
+    <method_context>
+      <method_credential group="root" user="root"/>
+    </method_context>
+    <exec_method name="start" type="method" exec="/opt/ooce/sbin/tailscaled" timeout_seconds="60"/>
```

Comment:
I'd prefer not to hardcode this path if possible... Are there good examples of how to template this kind of thing?

Reply:
Thanks for the pull request, we're all getting back to things after the holidays so sorry for the delay in looking at this. Somebody will be along to do a proper review in a day or two.

You can use variables in SMF manifests in conjunction with a call to `xform`; `openldap` is one example.
The `xform` function uses the same transformation variables as the `.mog` files so you can do something like this in the build.sh

```bash
XFORM_ARGS+="
    -DPREFIX=${PREFIX#/}
    -DPROG=$PROG
"

...

xform files/$PROG.xml > $TMPDIR/$PROG.xml
install_smf ooce $PROG.xml
```

Reply:
Thank you!!